salt/modules/xmpp.py

```diff
@@ -67,7 +75,7 @@ def __virtual__():
     """
     if HAS_LIBS:
         return __virtualname__
-    return (False, "Module xmpp: required libraries failed to load")
+    return False, "Module xmpp: required libraries failed to load"
 
 
 class SleekXMPPMUC(logging.Filter):
```

Comment:
Should this class also be deprecated so it can be renamed to something more generic?

Reply:
Both 3rd party libs have xmpp in them, so I did not see the need to rename the module. What name are you thinking?